### ISSUE TITLE
InstanceNormalization Layer

### ISSUE THREAD

**Issue by emi-research-dl** [keras-team/keras#19050]
Hi!!! Would be possible to add an InstanceNormalization layer to Keras?

Thanks in advanced!

**Comment by SuryanarayanaY** [comment]
Hi @emi-research-dl ,

Please refer [GroupNormalization](https://keras.io/api/layers/normalization_layers/group_normalization/) API where by setting the parameter `groups=-1` we can achieve InstanceNormalization.

Thanks!
